pages/mixin/mixin.pug

```diff
@@ -172,4 +172,56 @@ mixin orderDetailInfo(icon, text1, text2)
 	p
 		i.fa(class!=icon)
 		span=text1
-		span=text2
\ No newline at end of file
+		span=text2
+
+mixin addressForm(heading4, addMore, inputType)
+	div.address-form__content 
+		h3=heading4
+		a(href='#')=addMore
+		form
+			div.form-label
+				label(for='fname') Tên
+				label(for='lname') Họ & tên đệm
+				label(for='company') Công ty
+				label(for='address') Địa chỉ
+				label(for='city') Thành phố
+				label(for='nationality') Quốc tịch
+				label(for='postal') Postal/ Zip code
+				label(for='phone') Điện thoại
+			div.form-input
+				input(type='text' name='fname' value='Giang')
+				input(type='text' name='lname' value='Le Anh')
+				input(type='text' name='company' value='Công ty cổ phần Công nghệ DKT')
+				input(type='text' name='address' value='Tòa nhà Hà Nội Group, 442 Đội Cấn, Ba Đình, Hà Nội')
+				input(type!=inputType name='city' value='Hà Nội')
+				input(type!=inputType name='nationality' value='Việt Nam')
```

Comment:
mỗi cái label tương ứng 1 cái input luôn chứ c, c để thế này nguy cơ vỡ layout hoặc lệch lắm c

Reply:
c tạo mixin nhỏ thôi chị, mỗi cặp label input là 1 mixin nè, mình chỉ cần truyền name, type,... thôi. Chức k nên tạo mixin to như thế này, khó tái sử dụng

Reply:
okie e